micromanager_gui/_core_widgets/_presets_widget.py

```diff
@@ -38,6 +38,9 @@ def __init__(
 
         self._combo = QComboBox()
         self._combo.addItems(self._presets)
+        self._combo.setCurrentText(self._mmc.getCurrentConfig(self._group))
+        self._set_cfg_data_as_tool_tip(self._combo.currentText())
+        self._set_if_props_match_preset()
```

Comment:
same here... could this be connected to a change signal rather than called manually in "all the right places"?